verified-inference/host/api/service/tee/tee_attestation_service.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import HTTPException
+import socket
+from service.tee.entities import TeeAttestationResponse
+
+VSOCK_PORT = 5000
+GET_ATTESTATION_REQUEST = "GET_ATTESTATION_DOC"
+BUFFER_SIZE = 4096
+
+
+async def execute(enclave_cid: str) -> TeeAttestationResponse:
+    vsock = socket.socket(socket.AF_VSOCK, socket.SOCK_STREAM)
```

Comment:
Would be good to keep all low-level details related to `vsock` in a single component and not have it both here and in `proxy.py`